src/birthder.js

```diff
@@ -68,6 +68,40 @@
     }
   }
 
+  /**
+   * Create a room for all users except `username`
+   * @param {Robot} robot Hubot instance
+   * @param {string} username Birthday boy/girl
+   * @returns {Void}
+   */
+  const createBirthdayRoom = async (robot, username) => {
```

Comment:
The function creates `${username}-birthday-channel`, so name it `createBirthdayChannel`.